target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -426,6 +426,28 @@ define Device/tplink_tl-wdr4900-v2
 endef
 TARGET_DEVICES += tplink_tl-wdr4900-v2
 
+define Device/tplink_tl-wpa8630p-v2-de
```

Comment:
Consider a Device/tplink_tl-wpa8630p-v2_common
and inherit from it.

Reply:
I did not do this because the other TP-Link devices in here are all defined separately. Of course, I can define a common one?

Reply:
On a second thought: It not really worth it, just keep it as it is ...